### PR TITLE
pagination for table

### PR DESCRIPTION
**Pull request by seryozhabaleyko**

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[0111/095901.402:ERROR:directory_reader_win.cc(43)] FindFirstFile: ������� �� ������� ����� ��������� ����. (0x3)
```
Comment (saranchonkau) [inline debug.log]: remove it

File: package.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
     "@rollup/plugin-url": "^5.0.1",
     "@storybook/preset-scss": "^1.0.2",
     "@storybook/vue": "^6.0.21",
-    "@tager/admin-services": "^0.2.14",
+    "@tager/admin-services": "^0.2.15-rc.2",
```
Comment (saranchonkau) [inline package.json]: update to 0.2.15

File: src/components/DataTable/DataTable.vue [hunk 3]
```diff
@@ -79,11 +105,64 @@ export default defineComponent<Props>({
...
       handleSearchChange,
+      handlePageNumberChange,
+      handlePageSizeChange,
     };
   },
 });
 </script>
 
-<style scoped lang="scss"></style>
+<style>
```
Comment (saranchonkau) [inline src/components/DataTable/DataTable.vue]: remove this style

File: src/components/DataTable/DataTable.vue [hunk 4]
```diff
@@ -79,11 +105,64 @@ export default defineComponent<Props>({
...
+}
+
+.table-container {
+  height: 100%;
+  margin-bottom: 0;
+  flex: 1;
+
+  ::v-deep table tr:last-child td {
+    border-bottom: 0;
+  }
```
Comment (saranchonkau) [inline src/components/DataTable/DataTable.vue:142]: is it required? do we have some other way?

File: src/components/FormFieldColorInput/index.vue [hunk 5]
```diff
@@ -4,15 +4,23 @@
...
+import { defineComponent } from '@vue/composition-api';
+import FormFieldWrapper from '../FormFieldWrapper.vue';
+import ColorInput from '../ColorInput';
 
-export default Vue.extend({
+interface Props {
+  value: string;
+  name: { [key: string]: string };
+  label: { [key: string]: string };
+  error: { [key: string]: unknown };
```
Comment (saranchonkau) [inline src/components/FormFieldColorInput/index.vue]: are you sure it is correct?

File: src/components/FormFieldFileInput/index.vue [hunk 6]
```diff
@@ -12,31 +12,41 @@
...
+import FormGroup from '../FormGroup.vue';
+import FormFieldError from '../FormFieldError/index.vue';
+import InputLabel from '../InputLabel/index.vue';
+import FileInput from '../FileInput/index.vue';
 
-export default Vue.extend({
+interface Props {
+  name: string;
+  label: string;
+  value: [] | { [key: string]: unknown };
```
Comment (saranchonkau) [inline src/components/FormFieldFileInput/index.vue]: `value: Array<FileType> |  FileType | null`

File: src/components/FormFieldMultiSelect/index.vue [hunk 7]
```diff
@@ -14,63 +14,79 @@
...
-          typeof option === 'object' &&
-          'value' in option &&
-          'label' in option
-  )
+function isValidSelectOption(option: Option) {
+  return typeof option === 'object' && 'value' in option && 'label' in option;
 }
 
-export default Vue.extend({
+interface Option {
```
Comment (saranchonkau) [inline src/components/FormFieldMultiSelect/index.vue]: we already have type `OptionType`

File: typings/index.d.ts [hunk 8]
```diff
@@ -30,10 +30,13 @@ export type IconName =
   | 'search'
   | 'settings'
   | 'assignment'
+  | 'chevronLeft'
   | 'chevronRight'
   | 'description'
   | 'arrowDownward'
   | 'arrowUpward'
+  | 'arrowLeft'
+  | 'arrowRight'
```
Comment (saranchonkau) [inline typings/index.d.ts]: remove `arrowLeft` and `arrowRight`

File: package.json (hunk 2 above)
Comment (seryozhabaleyko) [reply package.json]: On npm this is the latest version.
image

File: src/components/DataTable/DataTable.vue (hunk 4 above)
Comment (seryozhabaleyko) [reply src/components/DataTable/DataTable.vue:142]: Yes, it is required. There is no other way at the moment.

File: src/components/FormFieldColorInput/index.vue (hunk 5 above)
Comment (seryozhabaleyko) [reply src/components/FormFieldColorInput/index.vue]: It's not necessary at all, I don't know how it got here.

File: package.json (hunk 2 above)
Comment (saranchonkau) [reply package.json]: @seryozhabaleyko 
image

Comment (seryozhabaleyko) [reply package.json]: 👌 done!

**Pull request merged**